pawn/Elements/Player/Commands/General.pwn

```diff
@@ -1125,19 +1126,6 @@ MinigameHelp:
             return 1;
         }
 
-        param_shift_int(iAmount);
```

Comment:
Without this line, you'll get a compile error on line 1129 because the iAmount variable does not exist. Please compile your code before submitting it?